lib/Runtime/Language/ModuleNamespace.cpp

```diff
@@ -338,28 +330,4 @@ namespace Js
         }
         return sortedExportedNames;
     }
-
-    Var ModuleNamespace::EntrySymbolIterator(RecyclableObject* function, CallInfo callInfo, ...)
```

Comment:
Does that affect the iteration order or does that continue to work because we already sorted them ?